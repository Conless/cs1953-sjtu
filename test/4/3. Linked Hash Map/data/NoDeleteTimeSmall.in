1
14 0 10 
0 1 ABegTPnoh25sQUqkd0Ez 
2 1 
2 1 
2 1 
2 1 
2 1 
0 13 OStIk4z9XuBq3Md8rUpe 
2 1 
0 2 bQRuwVNA9htLfB8Kdp3C 
2 13 
