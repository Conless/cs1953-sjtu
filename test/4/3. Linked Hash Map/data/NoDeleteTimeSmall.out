ABegTPnoh25sQUqkd0Ez 
ABegTPnoh25sQUqkd0Ez 
ABegTPnoh25sQUqkd0Ez 
ABegTPnoh25sQUqkd0Ez 
ABegTPnoh25sQUqkd0Ez 
ABegTPnoh25sQUqkd0Ez 
OStIk4z9XuBq3Md8rUpe 
1 ABegTPnoh25sQUqkd0Ez
13 OStIk4z9XuBq3Md8rUpe
2 bQRuwVNA9htLfB8Kdp3C
